central/policycategory/datastore/datastore_impl.go

```diff
@@ -127,6 +127,8 @@ func (ds *datastoreImpl) AddPolicyCategory(ctx context.Context, category *storag
 	if category.Id == "" {
 		category.Id = uuid.NewV4().String()
 	}
+	// Any category added after startup must be marked custom category.
+	category.IsDefault = false
```

Comment:
I think it would be better to just reject their request rather than mutating it?

Reply:
As discussed, we do this in policies - I went back and forth whether to throw error or mutate but then consistency won :)